tests/plotting/test_prediction_trajectory.py

```diff
@@ -143,6 +143,39 @@ def test_cross_entropy_smoke(prediction_trajectory: PredictionTrajectory):
     assert ce_stat.stats.shape == (3, 10)
 
 
+def test_rank_smoke(prediction_trajectory: PredictionTrajectory):
```

Comment:
Why is there a separate smoke and correctness test? This is probably not just for this PR, but it's general in the code base. It seems fine, but I'm curious.

Reply:
One tests on inputs with lots of different shapes but doesn't test the correctness of behavior (smoke) the other tests a single example to make sure it gets the correct answer.

Reply:
Aaaah makes sense, I missed the `@pytest.fixture(params=[(), (2, 2), (1,)])` in the fixture.